modules/300-prometheus/hooks/detect_vpa_max.go

```diff
@@ -88,10 +90,21 @@ func calculateNodesCapacity(input *go_hook.HookInput) error {
 	maxMem := resource.NewQuantity(totalPodsMemory/1, resource.BinarySI)
 	maxCPU := resource.NewMilliQuantity(totalPodsCPU/1, resource.DecimalSI)
 
-	// calculate longterm prometheus
+	if maxMem.Cmp(*minMem) == -1 {
+		maxMem = minMem
+	}
+
+	if maxCPU.Cmp(*minCPU) == -1 {
+		maxCPU = minCPU
+	}
+
+	// calculate longterm prometheus usage
 	maxLongtermMem := resource.NewQuantity(totalPodsMemory/3, resource.BinarySI)
 	maxLongtermCPU := resource.NewMilliQuantity(totalPodsCPU/3, resource.DecimalSI)
 
+	input.Values.Set("prometheus.internal.vpa.minMemory", minMem.String())
```

Comment:
What for do we need this value?

Reply:
@yalosev my bad, missed it from previous logic.